tsconfig.json

```
@@ -9,6 +9,7 @@
     "module": "ESNext",
     "moduleResolution": "Node",
     "baseUrl": ".",
+    "outDir": "./dist",
     "paths": {
       "~/*": ["*"]
```

shouldn't this be?

```suggestion
      "~/*": ["*"],
      "@divriots/starter-origami/*": ["*"]
```

looks like I'm the only one so far who configured this in my starter-kit, but not getting why so

---

:+1:

---

done